src/window.rs

```diff
@@ -1379,3 +1376,20 @@ impl Default for UserAttentionType {
         UserAttentionType::Informational
     }
 }
+
+/// A window level groups windows with respect to their z-position.
+///
+/// The relative ordering between windows in different window levels is fixed.
+/// The z-order of a window within the same window level may change dynamically on user interaction.
+///
+/// ## Platform-specific
+///
+/// - **iOS / Android / Web / Wayland:** Unsupported.
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+pub enum WindowLevel {
+    /// The window will always be below other windows,
+    AlwaysOnBottom,
+    Normal,
+    /// The window will always be on top of other windows,
+    AlwaysOnTop,
```

Comment:
```suggestion
    /// The window will always be below normal windows,
    AlwaysOnBottom,
    Normal,
    /// The window will always be on top of normal windows,
    AlwaysOnTop,
```

Modals, popups and other such windows will still be above on macOS